doc/sphinx/proof-engine/ltac.rst

```diff
@@ -1131,7 +1619,7 @@ Proving a subgoal as a separate lemma
          Use this feature at your own risk; building computationally relevant
          terms with tactics is fragile.
 
-   .. tacv:: transparent_abstract @ltac_expr using @ident
+   .. tacn:: transparent_abstract @ltac_expr3 using @ident
```

Comment:
Why is this a separate `tacn` instead of being a `{? using @ident }` clause in the one above?